### PR TITLE
fix for #4603 Quickstart install broke related to Camel-K

### PR DESCRIPTION
**Pull request by KurtStam**
Reusing our own dev scripts instead to minimize the chance this break over time.

https://github.com/syndesisio/syndesis/issues/4603

### CODE REVIEW

File: static/start [hunk 1]
```diff
@@ -1,281 +1,29 @@
...
-}
+read -p "This install will reset your current MiniShift. OK to continue? " -n 1 -r
+echo    # (optional) move to a new line
+if [[ $REPLY =~ ^[Yy]$ ]]
+then
 
+  wget https://github.com/syndesisio/syndesis/archive/master.zip
+  unzip ./master.zip 'syndesis-master/tools/bin/*'
+  if [ -d ".syndesis" ]; then
+    rm -fr .syndesis
```
Comment (rhuss) [inline static/start]: Please  note that ~/.syndesis is also used when installing the camel-k operator. So I dont think it should be unconditionally removed. Just remove the part which is affected by the scripts.

File: static/start [hunk 2]
```diff
@@ -1,281 +1,29 @@
...
-          return
-      fi
-    done
-}
+read -p "This install will reset your current MiniShift. OK to continue? " -n 1 -r
+echo    # (optional) move to a new line
+if [[ $REPLY =~ ^[Yy]$ ]]
+then
 
+  wget https://github.com/syndesisio/syndesis/archive/master.zip
```
Comment (rhuss) [inline static/start]: we use curl everywhere, should do it here, too imo.

File: static/start [hunk 3]
```diff
@@ -1,281 +1,29 @@
...
+  unzip ./master.zip 'syndesis-master/tools/bin/*'
```
Comment (rhuss) [inline static/start]: Best unzip in a temporary directory which you can remove later on. You can use mktemp for creating such a dir.

File: static/start (hunk 2 above)
Comment (KurtStam) [reply static/start]: done

File: static/start (hunk 3 above)
Comment (KurtStam) [reply static/start]: good idea - implemented in subsequent commit

**Pull request merged**